manifest.json

```diff
@@ -26,8 +26,7 @@
   "permissions": [
     "app.window.alwaysOnTop",
     {"fileSystem": ["write", "retainEntries"]},
-    "storage",
-    "https://www.google-analytics.com/"
+    "storage"
```

Comment:
keep trailing comma

Reply:
Any reason? https://google.github.io/styleguide/jsoncstyleguide.xml seems to avoid it (although it also doesn't seem to outright ban it either)

Reply:
I was going by the JS Object literals guide go/js-style#features-objects-use-trailing-comma. 
The JSON guide doesn't explicitly say one way or the other from what I could tell, but the examples omit it. Usually in these style guides the examples are not supposed to be normative, so I wouldn't use that as an indication that they should be omitted. Happy for you to do either.

Reply:
Ah good point, i think i'll omit it since there wasn't a trailing comma before and when in doubt i just like to match the codebase.